spidermon/contrib/scrapy/monitors.py

```diff
@@ -300,3 +327,17 @@ class SpiderCloseMonitorSuite(MonitorSuite):
         UnwantedHTTPCodesMonitor,
         FieldCoverageMonitor,
     ]
+
+
+class PeriodicMonitorSuite(MonitorSuite):
+    """This Monitor Suite implements the following monitors:
+
+        * :class:`PeriodicExecutionTimeMonitor`
+
+        You can easily enable this monitor *after* enabling Spidermon::
```

Comment:
**question** What you mean with *after* enabling Spidermon?

Reply:
I copied the docstring from SpiderCloseMonitorSuite. My understanding is that it's explaining that spidermon must be enabled for these standard monitors to be used.